.github/styles/Redocly-Docs/FutureTense.yml

```diff
@@ -1,6 +1,6 @@
 extends: existence
 message: 'Avoid using future tense: "%s". Use present tense instead.'
-link: https://intranet.redoc.ly/contributing/documentation-style-guide/#tone-and-audience
+link: https://intranet.redocly.com/contributing/documentation-style-guide/#tone-and-audience
```

Comment:
@lornajane could you check if the links are correct? None of them work for me, and the same goes for the neighbouring files.

Reply:
I guess we moved something? The correct link is https://intranet.redocly.com/content/documentation-style-guide#tone-and-audience